modules/security-context-constraints-about.adoc

```diff
@@ -243,12 +243,27 @@ pre-allocated values. Uses the minimum value of the first range as the default.
 Validates against the first ID in the first range.
 * `RunAsAny` - No default provided. Allows any `fsGroup` ID to be specified.
 
-ifndef::openshift-dedicated[]
 [id="authorization-controlling-volumes_{context}"]
+ifndef::openshift-dedicated[]
 == Controlling volumes
-
-The usage of specific volume types can be controlled by setting the `volumes`
-field of the SCC. The allowable values of this field correspond to the volume
+endif::openshift-dedicated[]
+ifdef::openshift-dedicated[]
+== Controlling volumes for CCS clusters
+endif::openshift-dedicated[]
+
+The usage of specific volume types 
+ifdef::openshift-dedicated[]
+for {product-title} with Customer Cloud Subscription (CCS) clusters 
+endif::openshift-dedicated[]
+can be controlled by setting the `volumes`
+field of the SCC. 
+ifdef::openshift-dedicated[]
+[NOTE]
+====
+In {product-title} deployments, you can set the `volumes` field of the SCC only for clusters that use the Customer Cloud Subscription (CCS) model. You cannot set the `volumes` field for {product-title} clusters that use a Red Hat cloud account, because this requires `cluster-admin` privileges.
```

Comment:
@wgordon17 can you please confirm if this statement is correct? Does setting the `volumes` field require cluster-admin privileges? Thanks!

Reply:
This is a `volumes` field on the SCC. So if you can create/edit an SCC, then you can use the `volumes` field. Since you can only create/edit an SCC if your `cluster-admin`, then it would apply to OSD CCS...however, including that here might be a weird fit. 

This seems like it should be a sub-heading under creating/editing SCCs ¯\_(ツ)_/¯ I get that it's under the "Understanding SCCs" heading, which is also true. If you want to _understand_ SCCs, you should probably know what the `volumes` field does.

I'll leave it up to you how best to convey this incredibly confusing scenario 😉

Reply:
@wgordon17 thank you for your comment. Following your comment and on further reflection, I have removed this note from the section. The CCS-specific heading for this volumes section I think speaks for itself, along with the note that I have added at the start of this page.

From the context of the requirement for OSDOCS-3196, my main aim is getting the OSD content parallel to the OCP and OSD content on this page, where applicable. I think the volumes subsection is ok here as a concept section. It fits with the other concept subsections in the "About security context constraints" section. I have also subsequently added a call out to the [Example security context constraints](http://file.fab.redhat.com/pneedle/pr48451/osd/managing-security-context-constraints.html) section to highlight the `volumes` parameter in the example.

If you feel that there is a need for a procedure section on controlling volumes through an SCC or to include a `volumes` parameter in the [Creating security context constraints for CCS clusters](http://file.fab.redhat.com/pneedle/pr48451/osd/managing-security-context-constraints.html#security-context-constraints-creating_configuring-internal-oauth) section also, please let me know. If that is the case, I will create a separate Jira to track that requirement separately. Thanks!